hw_5/task3.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+/* Create a function that takes a Roman numeral as its argument and return its value as a numeric decimal integer */
+
+/* I had to get to know debugger while doing this task and the task with factorial */
+
+const convertRomanToInteger = (romanString) => {
+  const romanNumbers = { I: 1, V: 5, X: 10, L: 50, C: 100, D: 500, M: 1000 };
+  let integer = 0;
+
+  for (let i = romanString.length - 1; i >= 0; i--) {
+    const current = romanString[i];
+    const next = romanString[i - 1];
```

Comment:
may be previous?

Reply:
it depends on which side to look)
for the user it will be previous, but for loop "for" it will be next)